src/stores/index.js

```diff
@@ -882,7 +942,14 @@ const Store = Reflux.createStore({
         null, // No error connecting.
         connectedDataService
       );
+
+      // Compass relies on `compass-connect` showing a progress
+      // bar, which is hidden after the instance information is loaded
+      // in another plugin.
+      this.StatusActions.showIndeterminateProgressBar();
```

Comment:
shouldn't this go in the other plugin? like `oneOfTheRegistry.on('connected', () => showTheBar)`

Reply:
maybe we would be able to avoid it after we load all the details?

Reply:
Yes - or it shouldn't be shown at all.
I'd like to defer that to this ticket if that's cool though: https://jira.mongodb.org/browse/COMPASS-4608